--- /etc/xen/scripts/network-bridge.old	2011-10-06 04:50:20.202630821 -0500
+++ /etc/xen/scripts/network-bridge	2011-10-06 04:51:18.166126405 -0500
@@ -115,6 +115,8 @@ do_ifup() {
             ip addr add ${addr_pfx} dev $1
         fi
         ip link set dev $1 up
-        [ -n "$gateway" ] && ip route add default via ${gateway}
+        if [ -n "$gateway" ]; then 
+            ip route add default via ${gateway}
+        fi
     fi
 }
